schema/deploy/shipping/views.sql

```diff
@@ -3624,6 +3625,34 @@ create or replace view shipping.__uw_priority_queue_v1 as (
         and screen_positive
     ),
 
+    -- Select added_surveillance_groups if they are in the group for today
+    added_surveillance_groups as (
+        select
+            redcap_url,
+            redcap_project_id,
+            redcap_record_id,
+            redcap_event_name,
+            redcap_repeat_instance,
+            encountered,
+            individual,
+            latest_invite_date,
+            latest_collection_date,
+            2 as priority,
+            'surveillance' as priority_reason,
+
+            latest_positive_hcov19_collection_date,
+            latest_prior_test_positive_date,
+            prior_test_positive_date_base,
+            alerts_off
+        from uw_enrollments
+        -- Filter to added_surveillance_groups for today.
+        -- To_Char(current_date, 'day') returns a string padded to 9 characters, so trim it.
+        -- Example: where 'tuesday' = 'tuesday'
+        where added_surveillance_groups = trim(To_Char(current_date, 'day'))
+        -- Filter for participants who haven't ever been invited or haven't been invited in 3 days
+        and (latest_invite_date is null or latest_invite_date < current_date - interval '3 days')
+	  ),
```

Comment:
Let's additionally filter out participants who had a sample collected within previous 3 days (see CTEs for positive attestation or normal surveillance for an example).